k8s.io/configmap-nginx.yaml

```diff
@@ -289,4 +289,13 @@ data:
           rewrite ^/.*$  https://prow.k8s.io/tide redirect;
         }
       }
+
+      server {
+        server_name youtube.k8s.io youtube.kubernetes.io;
```

Comment:
do you want to do `go.k8s.io/youtube` instead? (lines 229-248)

Reply:
youtube.k8s.io is actually shorter.

Reply:
This is a precedent setter though I guess, no? Was there a preferred format that I missed?

Reply:
Thinking about it -- I do kind of like the idea of `yt.k8s.io`/ `youtube.k8s.io` -- the default would go to the landing page for kubernetes, and it opens the door to adding more youtube specific redirects. I'm thinking something like:
`yt.k8s.io/sig-release` -> sig release playlist.

That winds up being a lot friendlier and easier to link to than this:
https://www.youtube.com/playlist?list=PL69nYSiGNLP3QKkOsDsO6A0Y1rhgP84iZ

Reply:
I'm down for that! That does make for a nice clean experience between various platforms (GitHub, YouTube, etc.). I have a to do list item for this.